snippets/visualbasic/VS_Snippets_CLR_System/system.Console.WriteLine/VB/tipcalc.vb

```diff
@@ -1,4 +1,4 @@
-﻿' <Snippet1>
+' <Snippet1>
 Public Class TipCalculator
    Private Const tipRate As Double = 0.18
    
```

Comment:
Lines 5-7 are legacy code. It's best to skip return codes, and simply have a `void Main()` method. Visual Basic console applications will also generally use modules rather than classes. 

But for each of these C# and Visual Basic samples, the Main method should:
- have the signature `static void Main()` or `Sub Main()` for entry points that don't depend on command-line arguments.
- have the signature `static void Main(string[] args)` or `Sub Main(args() As String)` for entry points that do depend on command-line arguments.

Reply:
Removed `Calculator` method and moved its code to `Main` & Changed `Class` to `Module` in commit 55d867c

Reply:
Changed C# main signature from `int Main` to `void Main` in commit a4ff83a